### ISSUE TITLE
No LoginModule found for org.keycloak.adapters.jaas.DirectAccessGrantsLoginModule

### ISSUE THREAD

**Issue by brusdev** [artemiscloud/activemq-artemis-operator#476]
The broker pod raises the following warning when keycloak login module is used:
```
[io.hawt.system.Authenticator] Login failed due to: No LoginModule found for org.keycloak.adapters.jaas.DirectAccessGrantsLoginModule
```

- Apply the following yaml:
```yaml
apiVersion: broker.amq.io/v1beta1
kind: ActiveMQArtemis
metadata:
  name: artemis-broker
spec:
  deploymentPlan:
    size: 1
    image: placeholder
---
apiVersion: broker.amq.io/v1beta1
kind: ActiveMQArtemisSecurity
metadata:
  name: keycloak-login
spec:
  loginModules:
    keycloakLoginModules:
    - name: 'keycloak-broker'
      moduleType: 'directAccess'
      configuration:
        realm: 'activemq'
        resource: 'artemis-broker'
        authServerUrl: 'http://KEYCLOAK_SVC_IP:8080/auth'
        useResourceRoleMappings: true
        principalAttribute: 'preferred_username'
        sslRequired: 'external'
        credentials:
        - key: 'secret'
          value: 'KEYCLOAK_CLIENT_SECRET'
  securityDomains:
    brokerDomain:
      name: "activemq"
      loginModules:
        - name: "keycloak-broker"
          flag: "sufficient"
```
Produce messages:
```
./amq-broker/bin/artemis producer --verbose --destination queue://test --user admin --password admin --url tcp://artemis-broker-ss-0.artemis-broker-hdls-svc.default.svc.cluster.local:61616
```

**Comment by gaohoward** [comment]
@brusdev  we probably need to use the latest init (cut  a new tag) to test it.

**Comment by brusdev** [comment]
@gaohoward I was thinking to add the Keycloack jars to activemq-artemis-broker-kubernetes-image by default as we did for the metrics plugin, see https://github.com/artemiscloud/activemq-artemis-broker-kubernetes-image/commit/1ea7bf9a579a6f294f945a85a090f9f0be7a87ba

**Comment by gtully** [comment]
it may also be a use case for extra mounts keycloak-jars-cp config map. Supporting a -cp postfix to mount a config map contents as subpaths in to the lib dir. maybe!

**Comment by brusdev** [comment]
The keycloak artifacts have been added in activemq-artemis-broker-kubernetes-image 1.0.14, see https://github.com/artemiscloud/activemq-artemis-broker-kubernetes-image/issues/48
Fixed in 1.0.8
